packages/11ty/_includes/components/figure/image.js

```diff
@@ -19,10 +19,14 @@ module.exports = function(eleventyConfig, globalData) {
   const { imageDir, figureLabelLocation } = globalData.config.params
 
   return async function({ alt='', canvasId, caption, choices, credit, id, iiifContent, label, manifestId, preset, src='' }) {
-    const imageSrc = path.join(imageDir, src)
+    // const imageSrc = path.join(imageDir, src)
+    const imageSrc = src.includes('http') || src.includes('https')
+      ? src
+      : path.join(imageDir, src)
     const labelElement = figurelabel({ caption, id, label })
     const srcParts = src.split(path.sep)
-    const hasTiles = srcParts[srcParts.length - 1] === 'tiles'
+    // const hasTiles = srcParts[srcParts.length - 1] === 'tiles'
+    const hasTiles = src && !src.match(/\.+(jpe?g|png|gif)/)
```

Comment:
This is still not an ideal way to check if a figure is intended to be displayed using image-service or <img>, as not having a common image file suffix does not guarantee that an image has tiles, but it works for demo purposes at the moment